fetch.bs

```diff
@@ -6718,6 +6753,415 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+   and  <a>allowed to use</a> the
+   <a>policy-controlled feature</a> "{{PermissionsPolicy/deferred-fetch-minimal}}"
+   <dd>
+    <p>512 kibibytes
+    <p class=note>The default of 640 kibibytes, decremented By
+    <a>quota reserved for <code>deferred-fetch-minimal</code></a>)
+
+
+   <dt><var>controlDocument</var>'s <a>node navigable</a>'s <a>navigable container</a>'s
+   <a>reserved deferred-fetch quota</a> is <a for="reserved deferred-fetch quota">normal quota</a>,
+   and <var>controlDocument</var> is <a>allowed to use</a> the <a>policy-controlled feature</a>
+   "{{PermissionsPolicy/deferred-fetch}}"
+   <dd><a for="reserved deferred-fetch quota">normal quota</a>
+
+   <dt><var>controlDocument</var>'s <a>node navigable</a>'s <a>navigable container</a>'s
+   <a>reserved deferred-fetch quota</a> is <a for="reserved deferred-fetch quota">minimal quota</a>,
+   and <var>controlDocument</var> is <a>allowed to use</a> the <a>policy-controlled feature</a>
+   "{{PermissionsPolicy/deferred-fetch-minimal}}"
+   <dd><a for="reserved deferred-fetch quota">minimal quota</a>
+  </dl>
+
+ <li>
+  <p><a for=list>For each</a> <var>navigable</var> in <var>controlDocument</var>'s
+  <a>node navigable</a>'s <a>descendant navigables</a> whose <a>container document</a>'s
+  <a>deferred-fetch control document</a> is <var>controlDocument</var>, decrement <var>quota</var>
+  by <var>navigable</var>'s <a>navigable container</a>'s <a>reserved deferred-fetch quota</a>.
+
+  <p class=note>Delegate some of the quota to nested documents that reserved it.
+
+ <li><p>If <var>quota</var> is equal or less than 0, then return 0.
+
+ <li><p>Let <var>quotaForRequestOrigin</var> be 64 kibibytes.
+
+ <li>
+  <p><a for=list>For each</a> <a>deferred fetch record</a> <var>deferredRecord</var> of
+  <var>controlDocument</var>'s <a>fetch group</a>'s
+  <a for="fetch group">deferred fetch records</a>:</p>
+
+  <ol>
+   <li><p>Let <var>requestLength</var> be the <a>total request length</a> of
+   <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>.
+
+   <li><p>Decrement <var>quota</var> by <var>requestLength</var>.
+
+   <li><p>If <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>'s
+   <a for=request>URL</a>'s <a for=url>origin</a> is <a>same origin</a> with <var>origin</var>,
+   then decrement <var>quotaForRequestOrigin</var> by <var>requestLength</var>.
+  </ol>
+
+ <li><p>If <var>quota</var> is equal or less than 0, then return 0.
+
+ <li><p>If <var>quota</var> is less than <var>quotaForRequestOrigin</var>, then return
+ <var>quota</var>.
+
+ <li><p>Return <var>quotaForRequestOrigin</var>.
+</ol>
+</div>
+
+<div algorithm>
+<p>To <dfn export>reserve deferred-fetch quota</dfn> for a <a>navigable container</a>
+<var>container</var> given an <a for=/>origin</a> <var>originToNavigateTo</var>:
+
+<p class=note>This is called when <var>container</var> and the document that initiated the
```

Comment:
```
This is called when container and the document that initiated the navigation (the "source document") are same origin.

It potentially reserves either 64kb or 8kb of quota for the frame, if it is not same origin with its parent and the permissions policy allow.
```

Could you please verify the triggering condition? The first sentence mentions same-origin only, while the second sentence still brings up `if it is not same origin with its parent`.

Even if the "not same origin" condition only applies to the 8kb case, when should it be triggered? Is it when different-origin container gets created or starts to navigate? But a different-origin container will get its initial policy copied from parent container, which it should not modifiy?